_data/other.yml

```
@@ -567,6 +567,14 @@ websites:
       exceptions:
           text: "Two factor authentication unavailable for 'End User' access, which is read-only."
 
+    - name: Weclapp 
+      url: https://*.weclapp.com/
```

To make the PR pass travis, please remove the *.
```suggestion
      url: https://weclapp.com/
```